changelogs/current.yaml

```diff
@@ -33,6 +33,12 @@ minor_behavior_changes:
   change: |
     When Lua script executes httpCall, backpressure is exercised when receiving body from downstream client. This behavior can be reverted
     by setting the runtime guard ``envoy.reloadable_features.lua_flow_control_while_http_call`` to false.
+- area: http
+  change: |
+    Modified the authority header value validator to allow the same characters as oghttp2
+    plus the "@" character. This is compliant with nghttp2, and supports the HTTP/1 use-cases
+    that allow user-info@ as part of the authority. This behavior can be reverted by setting
+    the runitme guard ``envoy.reloadable_features.internal_authority_header_validator`` to false.
```

Comment:
s/runitme/runtime/

Reply:
good catch.
I wonder why the docs are not being validated by the  spell checker.